allschool_test.go

```diff
@@ -65,9 +67,9 @@ func TestRemoveClass(t *testing.T) {
 
 	var v []openapi.ResponseMemberClass
 	decoder := json.NewDecoder(resp.Body)
-	_ = decoder.Decode(&v)
+	err = decoder.Decode(&v)
+
+	require.Nil(t, err)
 
-	assert.Equal(t, teachers[0], v[0].Owner.Id)
-	assert.Equal(t, members-1, len(v))
-	assert.Equal(t, classes[0], v[0].Id)
```

Comment:
It does not make sense to remove these, we want to make sure that the classes are what we expect them to be. Also createtestaccounts appears to never add 1 student to 2 classes(no repeat students) so a length of 0 may be right but I feel it makes for an unrealistic test. I also need to add students to the school classes.